<launch>


  <arg name="cameraIntrinsicInput"  default="$(find camera_transform_publisher)/camera_intrinsic.xml" />
  <arg name="cameraTopic"  default="/realsense/rgb" />

  <node pkg="camera_transform_publisher" type="aruco_publisher" name="aruco_publisher" output="screen">
  <!-- handeye_calib_camodocal arg pass -->
      <param name="cameraIntrinsicInput" type="str"  value="$(arg cameraIntrinsicInput)" />
      <param name="cameraTopic" type="str"  value="$(arg cameraTopic)" />
      <param name="tagSideLen"  type="double"  value="0.035" />
      <param name="planeSideLen"  type="double"  value="0.25" />
  </node>

</launch>
